pages/index.js

```diff
@@ -40,11 +41,20 @@ function Index({ trackId, backSideContent, latestId }) {
   }, [enterKey]);
 
   React.useEffect(() => {
+    if (localStorage.getItem("refresh") === "true"){
+      localStorage.setItem("refresh", false)
+      setRandomMsg(getRandomFromArray(RANDOM_MESSAGES))
+    } 
     ReactGA.initialize("UA-172416216-1");
     ReactGA.pageview(window.location.pathname + window.location.search);
     document.getElementById(
       "app"
     ).style.backgroundImage = `url('/images/${getRandomFromArray(BG_IMAGES)}')`;
+    window.onbeforeunload = function () {
+      console.log("reload")
+      localStorage.setItem("refresh", true)
+      return true;
```

Comment:
I think we need to remove this return to get rid of the dialog which appears when you try to refresh the page:

![cap](https://user-images.githubusercontent.com/24392180/198846914-7755b14b-3e8a-4dd4-9649-d95998ecc04d.png)

A quick google search led me to this page: https://stackoverflow.com/questions/17188058/using-onbeforeunload-without-dialog

Reply:
Fixed. Hopefully :)